instrumentation/kotlin-coroutines/kotlin-testing/kotlin-testing.gradle

```diff
@@ -3,6 +3,8 @@ apply from: "$rootDir/gradle/instrumentation.gradle"
 apply from: "$rootDir/gradle/test-with-kotlin.gradle"
```

Comment:
Do we need a separate testing project? Can we just merge this into kotlin-coroutines since these are tests for that instrumentation?

Reply:
Good call, this was just an artifact of how things were when this lived in java-concurrent.  Fixed and simplified.